app/config.js

```diff
@@ -17,12 +16,12 @@ const winCfg = new Config({
   }
 });
 
-let configDir = homedir();
-if (isDev) {
+let configDir = _paths.homeDir;
+if (_paths.isDev) {
```

Comment:
Maybe it could be better to abstract this "config path if we are in dev mode" in `config/paths`. Strange to move `isDev` in it and not this dev path mechanism.
isDev is an environnement notion, not a path one. maybe better to separate it from `paths`